source/overview/license-and-subscription.rst

```diff
@@ -71,13 +71,31 @@ You may incur retroactive charges for any unique users added that exceed the lic
 Renewing an Enterprise license
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The ability to renew your license in the Customer Portal will be available in a future release.
+You can renew your Mattermost license with a credit card if you have a standard Mattermost contract. When you renew your license, you can also increase the number of active users.
```

Comment:
I believe Cloud customers are charged automatically (auto-renew), and hence do not need to go through this process. We should document this in the FAQ.

Reply:
Added to the FAQ. 👍

Reply:
My apologies, re-reading the update, I think it makes sense to mention it here, to clarify the renewal.

Something like

```
You can renew your self-managed Mattermost Enterprise license with a credit card if you have a standard Mattermost contract. When you renew your license, you can also increase the number of active users. Y

If you use Mattermost Cloud, your subscription auto-renews so you don't need to follow this process.
```